app/create_file.py

```diff
@@ -1 +1,56 @@
-# write your code here
+from datetime import datetime
+from sys import argv
+from os import mkdir, chdir
+
+filename = ""
+directories = []
+
+if "-f" in argv:
+    index = 0
+
+    for _ in range(len(argv) - 1):
+        index += 1
+        if argv[_: _ + 1] == ["-f"]:
```

Comment:
1) Use some variable for index in arrays instead of _ : `argv[i : i+1]`
2) If you want to compare array element with some value, then use indexing instead of slice: `argv[i] == "-f"`